src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM_DISCOUNT = 50;
+  const NORMAL_TERM_DISCOUNT = 20;
+  const RENTAL_COST = 40;
+  const LONG_TERM = 7;
+  const NORMAL_TERM = 3;
+  const TOTAL_COST = days * RENTAL_COST;
```

Comment:
it's not a constant

Reply:
AI bot suggestions don't always need to be applied?